templates/zerver/integrations.html

```diff
@@ -2056,6 +2062,31 @@
 
     </div>
 
+    <div id="updown" class="integration-instructions">
+
+
+      <p>See Updown reports in Zulip! This is great for be up to date with
+      service's breaks you'd like to monitor!
+      </p>
+
+      <p>First, create the stream you'd like to use for updown, and subscribe
+      all interested parties to this stream. We recommend the
+      name <code>updown</code>.</p>
+
+      <p><code>{{ external_api_uri }}/v1/external/updown?api_key=abcdefgh&amp;stream=updown</code></p>
+
+      <p>Go to <a href="https://updown.io/settings/edit">Updown settings page</a>
+      and in <code>WEBHOOKS</code> section paste url we created above.
+      </p>
+
+      <p><b>Congratulations! You're done!</b><br />
+          Now you'll be updating about every problem with your service.
```

Comment:
Should be e.g. "Now you'll receive Updown notifications for your service in Zulip:"